dcicutils/secrets_utils.py

```diff
@@ -0,0 +1,53 @@
+import os
+import boto3
+import json
+from botocore.exceptions import ClientError
+from dcicutils.ecr_utils import CGAP_ECR_REGION
+
+
+def assume_identity():
+    """ Grabs application identity from the secrets manager.
+        Looks for environment variable IDENTITY, which should contain the name of
+        a secret in secretsmanager that is a JSON blob of core configuration information.
+        Default value is current value in the test account. This name should be the
+        name of the environment.
+    """
+    secret_name = os.environ.get('IDENTITY', 'dev/beanstalk/cgap-dev')
```

Comment:
Since we own the machine, I guess this is OK. I'm somewhat inclined to believe a better name like `AWS_IDENTITY` or even `AWS_SECRET_IDENTITY` is a better name for the env variable. A common name like `IDENTITY` may not be so secret. Clearly its protection is more how it's used and where it's available. But the name doesn't suggest the apparent semantics. I'm OK with just leaving it, but I wanted to say this out loud.